Tests/Rules/AvoidUnloadableModuleOrMissingRequiredFieldInManifest.tests.ps1

```diff
@@ -39,52 +39,52 @@ Describe "MissingRequiredFieldModuleManifest" {
 # Version number of this module.
 ModuleVersion = '1.0.0.0'
 '@
-            $violations[0].SuggestedCorrections[0].Text | Should Match $expectedText
-            Get-ExtentText $violations[0].SuggestedCorrections[0] $violationFilepath | Should Match ""
+            $violations[0].SuggestedCorrections[0].Text | Should -Match $expectedText
+            Get-ExtentText $violations[0].SuggestedCorrections[0] $violationFilepath | Should -Match ""
```

Comment:
isn't this `Should -BeNullOrEmpty`?

Reply:
Lol. Absolutely correct. As I said, I mainly applied some replace algorithms and checked at the end only that it corrects the right operators without looking too much at the tests. Will definitely fix that one